src/test/java/xtdb/api/XtdbHelloWorld.kt

```diff
@@ -10,7 +10,7 @@ class XtdbHelloWorld {
     @Test
     fun `hello world`() {
         Xtdb.openNode(Xtdb.Config(ServerConfig(0))).use { xtdb ->
-            DriverManager.getConnection("jdbc:postgresql://localhost:${xtdb.serverPort}/xtdb").use { conn ->
+            DriverManager.getConnection("jdbc:postgresql://localhost:${xtdb.serverPort}/xtdb?user=anonymous").use { conn ->
```

Comment:
Lots of places I now need to add `anonymous` explicitly as JDBC o/w passes some other user. A bit annoying, but not too much.

Reply:
hmm, this doesn't feel right - what user does it pass?